packs/actions/living-fortification.json

```diff
@@ -16,7 +16,39 @@
         "requirements": {
             "value": ""
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "ac",
+                "key": "RollOption",
+                "option": "living-fortification",
+                "suboptions": [
+                    {
+                        "label": "+1",
+                        "value": "1"
+                    },
+                    {
+                        "label": "+2",
+                        "value": "2"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "living-fortification"
+                ],
+                "selector": "ac",
+                "type": "circumstance",
+                "value": "{item|flags.pf2e.rulesSelections.livingFortification}"
+            },
```

Comment:
Let's make this an effect instead since it has a duration. You can basically copy the parry effect.